### PR TITLE
Test audio action of tracks controller

### LINKED ISSUE #85: Test TracksController#audio

### PR DESCRIPTION
**Pull request by robbevp**
fix #85

### CODE REVIEW

**Comment by robbevp** [comment]
Fun bug I couldn't figure out: if tests working with `TranscodedItem` failed, I got empty files like `["deleniti",` or `["ut",` in the app root. Oh well 🤷

**Review by charvp — CHANGES_REQUESTED** [review]
Instead of installing ffmpeg, the call should be mocked.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,53 +14,53 @@ jobs:
...
+            ${{ runner.os }}-gem-
+      - name: Install dependencies
+        run: |
+          bundle config set path vendor/bundle
+          bundle install --jobs 4 --retry 3
+      - name: Lint with rubocop
+        env:
+          RAILS_ENV: "test"
+        run: |
+          bundle exec rubocop -c .rubocop.yml
```
Comment (charvp) [inline .github/workflows/ci.yml:66]: These changes seem unnecessary and have merge conflicts.

**Review by charvp — APPROVED** [review]

**Review by charvp — CHANGES_REQUESTED** [review]
Tests fail in CI

**Review by charvp — APPROVED** [review]

**Pull request merged**